File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        //take user input
        n = get_int("Positive Numer: ");
    }
    while ( n < 1 || n > 8 );

    //print hashtags
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
                printf(" ");
            else
                printf ("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

#promt user for a positive integer between 1-8 included
while True:
    n = get_int("Insert number:")
    if n > 0 and n <= 8:
        break

#create loop for rows
for i in range(n):
    #create loop for columns
    for j in range(n):
        if (i + j) < n - 1:
            print(" ", end = "")
        else:
            print("#", end = "")
    print()
            

